src/test/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifierTestInt.java

```diff
@@ -0,0 +1,140 @@
+package jenkins.plugins.office365connector;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.when;
+import static org.powermock.api.mockito.PowerMockito.whenNew;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+
+import hudson.EnvVars;
+import hudson.model.AbstractBuild;
+import hudson.model.Job;
+import hudson.model.Run;
+import hudson.model.TaskListener;
+import jenkins.plugins.office365connector.helpers.ChangeLogSetBuilder;
+import jenkins.plugins.office365connector.helpers.ClassicDisplayURLProviderBuilder;
+import jenkins.plugins.office365connector.helpers.HttpWorkerAnswer;
+import jenkins.plugins.office365connector.helpers.WebhookBuilder;
+import org.apache.commons.io.IOUtils;
+import org.jenkinsci.plugins.displayurlapi.DisplayURLProvider;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Matchers;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+/**
+ * @author Damian Szczepanik (damianszczepanik@github)
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({DisplayURLProvider.class, Office365ConnectorWebhookNotifier.class, Run.class})
+public class Office365ConnectorWebhookNotifierTestInt {
+
+    public static final String JOB_NAME = "myFirstJob";
+    private static final int BUILD_NUMBER = 167;
+    private static final long START_TIME = 1508617305000L;
```

Comment:
here we have some date again so don't merge it as long as you are sure it does not block release process again :)